rust_src/README.md

```diff
@@ -48,10 +48,10 @@ use libc::{c_double, /* ... */ };
 // ...
 ```
 
-Second, `Lisp_Object` is an alias to `EmacsInt`:
+Second, `Lisp_Object` is a struct containing an EmacsInt:
 
 ```rust
-pub type Lisp_Object = EmacsInt;
+pub struct Lisp_Object(EmacsInt);
```

Comment:
Not true yet.

Reply:
Again, that's what I'm seeing in a fresh checkout.